### DISCUSSION TITLE
Show the install location of a package when using the "show" parameter

### DISCUSSION THREAD

**Discussion by ObiW4nKenobi** [microsoft/winget-cli#2636] [Ideas]
Please show the physical install path of the package files in a future update for the "show" parameter to the WinGet command. Maybe the installed size as well.

**Comment by sirtet** [comment]
There's an issue about that already: https://github.com/microsoft/winget-cli/issues/2298

**Comment by gavenkoa** [comment]
What is interesting `winget` knows about install location, because executables are properly linked to `%LOCALAPPDATA%\Microsoft\WinGet\Links`.
